codalab/model/bundle_model.py

```diff
@@ -1001,12 +1001,16 @@ def transition_bundle_running(self, bundle, worker_run, row, user_id, worker_id,
             'remote': worker_run.remote,
             'cpu_usage': cpu_usage,
             'memory_usage': memory_usage,
+            'data_size': worker_run.disk_utilization,
         }
 
-        # Increment user time as we go to ensure user doesn't go over time quota.
+        # Increment user time and disk as we go to ensure user doesn't go over quota.
         if user_id == self.root_user_id and hasattr(bundle.metadata, 'time'):
             time_increment = worker_run.container_time_total - bundle.metadata.time
             self.increment_user_time_used(bundle.owner_id, time_increment)
+        if hasattr(bundle.metadata, 'data_size'):
+            disk_increment = worker_run.disk_utilization - bundle.metadata.data_size
```

Comment:
Note that this line works because the worker_run.disk_utilization is the new bundle data size and bundle.metadata.data_size is the old data size. The bundle metadata data_size isn't updated until this line [here](https://github.com/codalab/codalab-worksheets/blob/fix/4298-check-running-disk-quota/codalab/model/bundle_model.py#L1032).